CONTRIBUTING.md

```diff
@@ -15,6 +19,32 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Data file management
+
+### Setup directory
+
+Each module should contain a `setup` directory that includes instructions and scripts to download and prepare any input data files used by the notebooks in the module.
+
+### Linking shared files 
+
+When trainings are run from the RStudio server (rstudio.ccdatalab.org), we store large input data files in the `/shared/data/training-modules` directory.
+The organization within this directory should mirror the arrangement of the repository, so that we can easily link files and folders from this shared directory to mirrored paths within a clone of this repository.
+
+Linking files from the shared directory to a cloned repository is done with the `scripts/link-data.sh` bash script, so this script should be kept up to date as any needed files are added to the `/shared/data/training-modules` directory.
+When possible, link to enclosing directories rather than individual files to keep links simpler, but see an important caveat below.
+
+Because this script is also used to set up directories for training, the links should not include _all_ files needed for _every_ notebook: 
+- Files that are created during a training module should not  be included in this script
+- Directories that users will need to write to should not be links, or the user will not be able to write their own files.
+
+
```

Comment:
How do I get the files I need to perform development? Do I run the `scripts/link-data.sh` or do I get them from S3?

Reply:
Probably the former, but I was trying not to be too prescriptive, in part because developing locally/in Docker could be a different process. I can imagine a situation where you might start with local copies, then move them to `/shared` and add links. 

Which is maybe to say that I haven't solved this question for everyone yet?

Reply:
Could an interim solution be some suggestions about multiple paths (as in series of steps, not file paths!) one could take depending on the situation? Rather than couch it as "here's exactly what to do." (Might add something about telling reviewers how to test in PRs, too.)

You may be able to tell that I am imagining a situation where I emerge from a meeting about something else entirely and want to run something I am reviewing myself. 😅

Reply:
I would also find an issue to track this request + merging this PR to be acceptable.

Reply:
I might go for this option!